launch/llmctl/src/main.rs

```diff
@@ -321,7 +323,9 @@ async fn list_single_model(
     );
 
     let mut models = Vec::new();
-    let etcd_client = distributed.etcd_client();
+    let etcd_client = distributed
```

Comment:
Why do we test this at several places in this file? Wouldn't checking for the etcd_client on distributedruntime once be enough?
I don't think there should be multiple entrypoints to this main.rs?

Reply:
I'm not sure, I just had to adapt this file to keep it working. Possibly we should refactor later, but really `llmctl` shouldn't exist, it's a hack to do discovery manually for some Python demos.